index.js

```diff
@@ -38,11 +37,10 @@ function getPseudoISO (date, timeZone) {
     second: '2-digit',
     hour12: false,
     timeZone
-  }, 'af-ZA', timeZone).format(date)
-  if (str.length !== 19 || str[10] !== ' ') {
-    throw new Error('Pseudo-ISO failed, got: ' + str)
-  }
-  return str.replace(' ', 'T') + 'Z'
+  }, 'en-US', timeZone).formatToParts(date)
+  const find = (key) => formattedParts.find(part => part.type === key).value
+  const utcDate = `${find('year')}-${find('month')}-${find('day')}T${find('hour')}:${find('minute')}:${find('second')}Z`
+  return new Date(utcDate)
```

Comment:
It used to return a string, but now it returns a `Date` instance?

Reply:
True, oops! The error wasn't apparent since it got `new Date`'d in `getOffset`.